R/tm_a_regression.R

```diff
@@ -37,9 +37,9 @@
 #'     It takes the form of `c(value, min, max)` and it is passed to the `value_min_max`
 #'     argument in `teal.widgets::optionalSliderInputValMinMax`.
 #'
-# nolint start: line_length.
+# nolint start: line_length
```

Comment:
As discussed, I don't think this is correct and the period is necessary

Note that on the first screenshot, I removed the period and changed the linter name and it passes.

On the second it fails

<img width="996" height="517" alt="Image" src="https://github.com/user-attachments/assets/b6be12b5-6758-4fae-b2a2-7dac890f7142" />

<img width="977" height="490" alt="Image" src="https://github.com/user-attachments/assets/ab85a775-6920-43fd-8bee-b1809c9bdeb7" />